yt/utilities/lib/element_mappings.pyx

```diff
@@ -95,6 +95,24 @@ cdef class ElementSampler:
     cdef int check_inside(self, double* mapped_coord) nogil:
         pass
 
+    @cython.boundscheck(False)
+    @cython.wraparound(False)
+    @cython.cdivision(True)
+    def check_contains(self, np.float64_t[:,::1] vertices,
+                             np.float64_t[:,::1] positions):
+        cdef np.ndarray[np.float64_t, ndim=2] mapped_coords
+        cdef np.ndarray[np.uint8_t, ndim=1] mask
+        mapped_coords = self.map_reals_to_unit(vertices, positions)
+        mask = np.zeros(mapped_coords.shape[0])
```

Comment:
I was getting a buffer type mismatch since np.zeros defaults to float but mask is declared as int, had to add:
```suggestion
        mask = np.zeros(mapped_coords.shape[0], dtype=np.uint8) 
```